webknossos/webknossos/dataset/layer.py

```diff
@@ -230,7 +230,10 @@ def _dump_mag_path(path: Path, dataset_path: Path) -> str:
     path = resolve_if_fs_path(path)
     if str(path).startswith(str(dataset_path)):
         return str(path).removeprefix(str(dataset_path)).lstrip("/")
-    if path.is_relative_to(dataset_path):
+    if (
```

Comment:
Can you extract that into a `safe_is_relative_to` function in `webknossos/utils.py`?